src/status_im/contexts/communities/events.cljs

```diff
@@ -248,3 +250,14 @@
                                  {:error   err
                                   :chat-id chat-id
                                   :event   "share-community-channel-url-with-data"}))}]})))
+(rf/reg-event-fx :communities/set-addresses-for-permissions
```

Comment:
Please add an empty line here

Reply:
also please can you elaborate, how this change related to current pr?

Reply:
Sorry messed thing up when solving conflicts, Reverting